server/database.js

```diff
@@ -81,6 +81,7 @@ class Database {
         "patch-monitor-oauth-cc.sql": true,
         "patch-add-timeout-monitor.sql": true,
         "patch-add-gamedig-given-port.sql": true,
+        "patch-status-page-locale-selector.sql": true,
```

Comment:
It is deprecated. Need to use knex migrations

https://github.com/louislam/uptime-kuma/tree/master/db/knex_migrations

Reply:
Done

Reply:
It seems the test does not agree with me, has it already been updated to work with these knex migrations?